services/parser/parser.go

```diff
@@ -115,10 +132,12 @@ func GetBlocksIntervalToFetch(params Params, ctx context.Context) (int64, int64,
 		lastParsedBlock = currentBlock - int64(params.BacklogCount)
 	}
 
-	if currentBlock-lastParsedBlock > params.MaxBacklogBlocks {
-		lastParsedBlock = currentBlock - params.MaxBacklogBlocks
+	nextParsedBlock := lastParsedBlock + params.BlocksPerRound
```

Comment:
can we test this too?

Reply:
we use integration tests for parser

Reply:
Where is the test for this? I see change but there is no tests that will cover this case